mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml

```diff
@@ -323,6 +348,53 @@
 
 </section>
 
+<section anchor="ip-flow-id-info"
+                 title="Management and Control Information Summary">
+          <t>
+            The following summarizes the set of information that is needed to
+            identify individual and aggregated DetNet flows:

```

Comment:
configure DetNet MPLS over IP

Reply:
Add the label (S-label or F-label) to be mapped to UDP/IP
Destination UDP port (full 6-tuple information needed)
No ranges/prefixes/wildcards needed.